jetty-ee10/jetty-ee10-osgi/test-jetty-ee10-osgi/pom.xml

```diff
@@ -535,12 +552,6 @@
               <!-- pax.exam.system>default</pax.exam.system -->
             </systemPropertyVariables>
             <argLine>-Dconscrypt-version=${conscrypt.version}</argLine>
-            <!-- TODO -->
```

Comment:
simply use the junit4 annotation for those tests.
Ideally I'd like to have those tests back but that's more and not related to the changes here.